### PR TITLE
Fake PR to highlight examples of unhelpful comments

### PR DESCRIPTION
**Pull request by sandyaaaa**

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -105,11 +105,11 @@ var Error = React.createClass({
 var Main = React.createClass({
   getInitialState: function() {
     return {topVideo: [],
-            videos: [],
-            navItems: [],
-            currentChannel: '927',
-            pages: [],
-            loading: false,
+            videos: [],
```
Comment (ssnkr) [inline js/script.js]: extra space

File: js/script.js [hunk 2]
```diff
@@ -105,11 +105,11 @@ var Error = React.createClass({
 var Main = React.createClass({
   getInitialState: function() {
     return {topVideo: [],
-            videos: [],
-            navItems: [],
-            currentChannel: '927',
-            pages: [],
-            loading: false,
+            videos: [], 
+            navItems: [],
```
Comment (ssnkr) [inline js/script.js]: extra space

File: js/script.js [hunk 3]
```diff
@@ -105,11 +105,11 @@ var Error = React.createClass({
...
   getInitialState: function() {
     return {topVideo: [],
-            videos: [],
-            navItems: [],
-            currentChannel: '927',
-            pages: [],
-            loading: false,
+            videos: [], 
+            navItems: [], 
+            currentChannel: '927',
```
Comment (ssnkr) [inline js/script.js]: extra space

File: js/script.js [hunk 4]
```diff
@@ -105,11 +105,11 @@ var Error = React.createClass({
...
-            videos: [],
-            navItems: [],
-            currentChannel: '927',
-            pages: [],
-            loading: false,
+            videos: [], 
+            navItems: [], 
+            currentChannel: '927', 
+            pages: [], 
+            loading: false,
```
Comment (ssnkr) [inline js/script.js]: extra space

**Comment by ssnkr** [comment]
Looks like you checked in some trailing spaces on several lines of your change set. Our style guide specifies no trailing whitespace. Can you take a look at this?

**Comment by ssnkr** [comment]
Your code violates the style guide in several places. Can you check for linting errors and fix them?

**Comment by sandyaaaa** [comment]
Sorry, what do you mean by linting errors?

**Comment by ssnkr** [comment]
Linting?! It's strange you don't know this given your experience.

**Comment by ssnkr** [comment]
Linter? You should know this by now given your experience. Can you just look it up? thanks

File: js/script.js (hunk 1 above)
Comment (ssnkr) [inline js/script.js]: 🤢

**Comment by ssnkr** [comment]
Looks great! Can you please rebase onto master before you merge this?

**Comment by ssnkr** [comment]
Did you even test this code before checking it in? 😂

**Pull request merged**